stable/concourse/values.yaml

```diff
@@ -1097,6 +1097,21 @@ concourse:
 ##
 web:
 
+  ## Create HorizontalPodAutoscaler object.
+  ##
+  # autoscaling:
```

Comment:
We're keeping these options not using comments in order to parse those, and going with an `enabled` basis would be more consistent with the [reviewing guidelines](https://github.com/helm/charts/blob/ef91306195f2a2cb89349c2729bb8c30065a368b/REVIEW_GUIDELINES.md#autoscaling--horizontalpodautoscaler).